modules/llm-cache/ds/config.h

```diff
@@ -53,16 +53,29 @@ struct FileCacheConfig : public KVCacheConfig {
   int splitNumber;
   std::string root;
   FilesystemType filesystemType;
+  int clientGCInterval;  // second
+  int ttl;               // second
+  bool enbaleGlobalGC;
+  int globalGCInterval;  // second
+  int globalTTL;         // second
 
   FileCacheConfig(int tensorByte = 10, int cacheCapacity = 10, int layer = 1,
                   int batchSize = 4, int splitNumber = 2,
                   std::string root = "/tmp/llm_cache/",
-                  FilesystemType filesystemType = LOCAL)
+                  FilesystemType filesystemType = LOCAL,
+                  int clientGCInterval = 3, int ttl = 5,
+                  bool enbaleGlobalGC = false, int globalGCInterval = 30,
+                  int globalTTL = 50)
```

Comment:
TTL is too short. You may use 30 minutes and 3 hours for local and global, respectively.

As well as the interval.

Reply:
Done